docs/src/main/asciidoc/logging.adoc

```diff
@@ -217,7 +218,7 @@ quarkus.log.category."org.hibernate".level=DEBUG
 ----
 
 When you set the log level to below `DEBUG`, you must also adjust the minimum log level.
-This setting is either global, using the `quarkus.log.min-level` configuration property, or per category:
+You can apply this setting either globally with the `quarkus.log.min-level` configuration property, or per category:
```

Comment:
Are we doing second-person here? Maybe instead:
```suggestion
This setting may be applied either globally with the `quarkus.log.min-level` configuration property, or per category:
```

Reply:
No, in this case keeping `may` is appropriate I think; I would use `might` for something that could happen or is allowed to happen out of the user's control (but is not certain), but use `may` for something that the user is specifically being permitted or suggested to do.